### ISSUE TITLE
No typescript types found

### ISSUE THREAD

**Issue by jake8655** [C-o-d-e-C-o-w-b-o-y/react-snow-overlay#4]
## Issue
With dd4bb1f9482c055e9b2bbccfc573d9074071c4ba having removed the `types` key in `exports` the typescript compiler doesn't know where the types are. The following error is thrown:
```
Could not find a declaration file for module 'react-snow-overlay'. '/home/jake/Desktop/projects/website/node_modules/react-snow-overlay/packages/react-snow-overlay/dist/index.es.js' implicitly has an 'any' type.
  There are types at '/home/jake/Desktop/projects/website/node_modules/react-snow-overlay/packages/react-snow-overlay/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports". The 'react-snow-overlay' library may need to update its package.json or typings.
```

## Solution
Revert the commit? I don't know what warning it caused on build but it might be worth it to investigate how it's supposed to be defined because without dd4bb1f9482c055e9b2bbccfc573d9074071c4ba it works flawlessly

**Comment by C-o-d-e-C-o-w-b-o-y** [comment]
Thanks a lot for flagging @jake8655 ! Weird that [this line](https://github.com/C-o-d-e-C-o-w-b-o-y/react-snow-overlay/blob/main/package.json#L16) doesn't work

This has now been [reverted](https://github.com/C-o-d-e-C-o-w-b-o-y/react-snow-overlay/commit/8f171e847a8ef00a085bfb514dbf03abb01d0d61) and published as version `1.0.12`

**Comment by jake8655** [comment]
Thank you for your quick responses and great work @C-o-d-e-C-o-w-b-o-y !
